tests/eas/preliminary.py

```diff
@@ -301,6 +301,9 @@ def test_asym_cpucapacity(self):
         - Finally, revert to the old cpu_capacity values and check the flag
           returns to its old value.
         """
+        if not self.target.file_exists(self._get_cpu_cap_path(0)):
```

Comment:
Can it make sense also to cross-check with the specific kernel version.
I guess that starting from a certain version we expect this sysfs attribute to be exposed... otherwise the test should not be skipped by instead FAILS, isn't it?